Library/ViewModels/MessagesViewModelTests.swift

```diff
@@ -240,7 +240,7 @@ internal final class MessagesViewModelTests: TestCase {
       self.scheduler.advance()
 
       self.messages.assertValueCount(1)
-      self.replyButtonIsEnabled.assertValues([false, false, true])
+      self.replyButtonIsEnabled.assertValues([false, true, true])
```

Comment:
This change is because the user template now has the `isBlocked` explicitly set, so the `isBlocked == false` check causes true to be emitted instead. (I also changed isBlocked to be required to avoid weirdness like this in the future; optional bools are weird, especially when we expect them to actually exist.)

Reply:
This comment would also be good in the code!